tests/expected/given_config.yaml

```diff
@@ -28,6 +29,7 @@ postgres:
       table: auto_table
       srid: 4326
       geometry_column: geom
+      id_column: feat_id
```

Comment:
the `feat_id: int4` line in the `properties` section on line 43 should not be there -- it indicates that the same value from the `feat_id` column will be included twice - once as the feature ID, and again as a regular propery.